Engine/src/Engine/SceneBuilder/Entity.cpp

```diff
@@ -2,7 +2,16 @@
 
 namespace Engine
 {
+    int Entity::s_totalEntities = 0;
+
+    int Entity::getTotalEntities()
+    {
+        return s_totalEntities;
```

Comment:
if the intention is to get the number of entities, we can just call m_registry.size();

Reply:
fixed